src/core/electric_utility.jl

```diff
@@ -195,6 +200,51 @@ struct ElectricUtility
         cambium_emissions_region = "NA - Cambium data not used for climate emissions" # will be overwritten if Cambium is used
         
         if !is_MPC
+            # Initialize clean energy fraction series
```

Comment:
We'll want a check that user-provided values are between 0-1 (inclusive), and throw an error if they are not

Reply:
Added checks for user provided values